### PR TITLE
[Tools]: Sample Dev go console

### PR DESCRIPTION
**Pull request by Milerius**
## Description

CLI dev console with the following features:

- Creation of a wallet
- Loading of a wallet
- Deletion of a wallet
- Dump address of the loaded wallet
- Help

## How to test

Follow README instructions

## Types of changes

Side project

## Checklist

- [x] Create pull request as draft initially, unless its complete.
- [x] Add tests to cover changes as needed.
- [x] Update documentation as needed.
- [x] If there is a related Issue, mention it in the description.

If you're adding a new blockchain

- [x] I have read the [guidelines](https://developer.trustwallet.com/wallet-core/newblockchain#integration-criteria) for adding a new blockchain.

### CODE REVIEW

**Comment by hewigovens** [comment]
can you move this to `samples/go`and adjust folder structures a little bit

**Comment by Milerius** [comment]
> can you move this to `samples/go`and adjust folder structures a little bit, I will move `walletconsole` to `cpp` folder later

Sure

**Comment by catenocrypt** [comment]
To be safe, I suggest adding a warning message at startup: `This is a test tool, DO NOT USE WITH REAL FUNDS!`

**Comment by catenocrypt** [comment]
Very nice tool.
I have some doubts how to make sure it will compile in the future (doesn't get borked)

**Review by hewigovens — DISMISSED** [review]
We can build this on `linux-sampleapp-ci`

**Review by catenocrypt — DISMISSED** [review]

**Comment by hewigovens** [comment]
@Milerius can you build it on CI? let's merge it

**Review by hewigovens — APPROVED** [review]

File: registry.json [hunk 1]
```diff
@@ -400,6 +400,7 @@
     "coinId": 118,
     "symbol": "ATOM",
     "decimals": 6,
+    "chainId": "cosmoshub-4",
```
Comment (catenocrypt) [inline registry.json:403]: I think changes do not belong to this PR

**Review by catenocrypt — DISMISSED** [review]
I think there is some merge/rebase issue with this PR currently, many unrelated (from previous PRs?) show up as changes

**Review by hewigovens — APPROVED** [review]
